src/main/java/net/dv8tion/jda/api/events/guild/override/GenericPermissionOverrideEvent.java

```diff
@@ -24,7 +24,7 @@
 import javax.annotation.Nullable;
 
 /**
- * Indicates that a {@link net.dv8tion.jda.api.entities.GuildChannel GuildChannel's} {@link PermissionOverride} was created/deleted/updated.
+ * Indicates that a {@link PermissionOverride}  for a {@link net.dv8tion.jda.api.entities.GuildChannel GuildChannel} was created, deleted, or updated.
```

Comment:
An extra space was introduced here.
```suggestion
 * Indicates that a {@link PermissionOverride} for a {@link net.dv8tion.jda.api.entities.GuildChannel GuildChannel} was created, deleted, or updated.
```